### DISCUSSION TITLE
Checklist Enhancement

### DISCUSSION THREAD

**Discussion by pcpsadmin** [IBM/mac-ibm-notifications#84] [Ideas]
It would be nice to be able to flag checkboxes to be checked by default. I'm looking to utilize Notifier in a specific use case, where the state of the checkboxes can persist between executions.

For example, present user with a popup that contains 2 checkboxes. They check option 1, but leave option 2 empty. Script could write a plist with the user's chosen checkboxes states. On the next popup, script could read from the plist and flag each checkbox as either on or off (1 or 0), etc.

Obviously I'm not looking for Notifier to create/read the plist as I'd write that into the script myself. Just looking for a way to flag whether a checkbox should be checked or not before the popup is displayed.

**Comment by SMartorelli** [comment]
Hi @pcpsadmin thanks for your feature request. Yes could be really useful for the workflow you described so I'm going to create a FR issue from this discussion so you can track the development of this feature in the next version of IBM Notifier.
